data/json/items/armor.json

```diff
@@ -9610,5 +9645,39 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "unitard",
+    "type": "ARMOR",
+    "name": "unitard",
+    "description": "A skin-tight spandex garment resembling a crew necked leotard combined with leggings.   Commonly used by gymnasts, dancers and equestrian vaulters, the unitard provides overall coverage with great flexibility.",
+    "weight": "110 g",
+    "volume": "250 ml",
+    "price": 3100,
+    "material": [ "lycra" ],
+    "symbol": "[",
+    "color": "dark_gray",
+    "covers": [ "TORSO", "LEGS" ],
+    "coverage": 90,
+    "warmth": 20,
+    "material_thickness": 1,
+    "flags": [ "VARSIZE", "SKINTIGHT", "WATER_FRIENDLY" ]
+  },
+  {
+    "id": "zentai",
+    "type": "ARMOR",
+    "name": "zentai",
+    "description": "Short for /zenshin taitsu/, the zentai is a form fitting full body nylon and spandex garment.  Though it covers your face, you can see through the integrated hood.",
+    "weight": "300 g",
+    "volume": "500 ml",
+    "price": 2500,
+    "material": [ "lycra" ],
+    "symbol": "[",
+    "color": "dark_gray",
+    "covers": [ "HEAD", "MOUTH", "ET", "TORSO", "ARMS", "HANDS", "LEGS", "FEET" ],
```

Comment:
What is "ET" here?

Reply:
Zentai covers aliens, nothing unusual.

Reply:
oh, dammit. I thought I fixed that. I always mess up during fixing merge conflicts.